kubernetes/cray-s3/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 name: cray-s3
 description: Exposes Ceph Rados gateways
-version: 0.2.0
+version: 0.4.0
```

Comment:
Was the change that was made backwards-compatible? Would it work on an older version of csm?